graph/pull_v2.go

```diff
@@ -58,7 +59,13 @@ func (p *v2Puller) pullV2Repository(tag string) (err error) {
 		taggedName = utils.ImageReference(p.repoInfo.LocalName, tag)
 	} else {
 		var err error
-		tags, err = p.repo.Manifests().Tags()
+
+		manSvc, err := p.repo.Manifests(context.Background())
```

Comment:
@stevvooe @dmcgowan so FYI in many other places, we use a nil context, not sure if that's a problem.